migrations/versions/0fb7d9d2fcd6_assosiating_a_list_of_books_to_user.py

```
@@ -0,0 +1,30 @@
+"""assosiating a list of books to user
+
+Revision ID: 0fb7d9d2fcd6
+Revises: c97f085a1d93
+Create Date: 2024-12-26 17:08:00.998336
+
+"""
+from typing import Sequence, Union
+import sqlmodel
```

Ruff (0.8.2)

`sqlmodel` imported but unused

Remove unused import: `sqlmodel`

(F401)